include/ur.py

```diff
@@ -1304,6 +1310,59 @@ def __str__(self):
         return str(ur_usm_pool_info_v(self.value))
 
 
+###############################################################################
+## @brief Virtual memory granularity info
+class ur_virtual_mem_granularity_info_v(IntEnum):
+    MINIMUM = 0x30100                               ## [size_t] size in bytes of the minimum virtual memory granularity.
+    RECOMMENDED = 0x30101                           ## [size_t] size in bytes of the recommended virtual memory granularity.
+
+class ur_virtual_mem_granularity_info_t(c_int):
+    def __str__(self):
+        return str(ur_virtual_mem_granularity_info_v(self.value))
+
+
+###############################################################################
+## @brief Virtual memory access mode flags.
+class ur_virtual_mem_access_flags_v(IntEnum):
+    READ_WRITE = UR_BIT(0)                          ## Virtual memory both read and write accessible
```

Comment:
Sorry for the late reaction to this, but I just realized that we are losing the ability to set the access mode to "none" here by defining `READ_WRITE` as 0. CUDA considers an access mode of 0 as revoking access. @kbenzie - Could we either have a `NONE` value here or offset the existing ones?

Reply:
Missed that! Yeah, I'll add none.